apps/heureka-next/src/components/Services/ServicesList/ServicesList.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from "react"
+import { DataGrid, DataGridRow, DataGridHeadCell, Pagination, Message } from "@cloudoperators/juno-ui-components"
+import { ServiceListItem } from "./ServiceListItem"
+import { FilterSettings } from "../../common/Filters/types"
+import { useFetchServices } from "../useFetchServices"
+import { EmptyDataGridRow } from "../../common/EmptyDataGridRow/EmptyDataGridRow"
+
+const COLUMN_SPAN = 6
+
+type ServiceListProps = {
+  filterSettings: FilterSettings
+}
+
+export const ServicesList = ({ filterSettings }: ServiceListProps) => {
+  const { loading, error, services, currentPage, totalNumberOfPages, goToPage } = useFetchServices({
+    filterSettings,
+  })
+
+  return (
+    <div className="flex-1 flex flex-col gap-10 overflow-hidden">
+      <div className="overflow-scroll">
+        <DataGrid minContentColumns={[3, 4]} columns={COLUMN_SPAN} className="clickableTable">
```

Comment:
Where is this class `clickableTable ` defined? Just curious...

Reply:
actually this is not required, so I removed it.